aspnetcore/migration/20_21.md

```diff
@@ -28,20 +28,19 @@ This article provides an overview on migration to  ASP.NET Core 2.1. It does not
 
 ## Update the project file to use 2.1 versions
 
-Update the *.csproj* project file:
+Update the project file:
 
-* Change `<TargetFramework>netcoreapp2.0</TargetFramework>` to the 2.1 version, that is `<TargetFramework>netcoreapp2.1</TargetFramework>`.
-* Replace the version specified "Microsoft.AspNetCore.All" package reference with the versionless "Microsoft.AspNetCore.App" package reference. You may need to add dependencies that were removed from "Microsoft.AspNetCore.All". See [Migrating from Microsoft.AspNetCore.All to Microsoft.AspNetCore.App](xref:fundamentals/metapackage#migrate) and [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). If you're targetting the .NET Framework:
-
-  * Add individual package references instead of a meta package reference.
+* Change the target framework to .NET Core 2.1 by updating the project file to `<TargetFramework>netcoreapp2.1</TargetFramework>`.
+* Replace the package reference for `Microsoft.AspNetCore.All` with a package reference for `Microsoft.AspNetCore.App`. You may need to add dependencies that were removed from `Microsoft.AspNetCore.All`. For more information, see <xref:fundamentals/metapackage#migrate> and <xref:fundamentals/metapackage-app>. For apps that target the .NET Framework:
+  * Add individual package references instead of a metapackage reference.
   * Update each package reference to 2.1.
-* Remove all references to `<DotNetCliToolReference>` elements for "Microsoft.AspNetCore", "Microsoft.VisualStudio", and "Microsoft.EntityFrameworkCore" packages. These tools have been replaced by global tools.
+* Remove all references to **&lt;DotNetCliToolReference&gt;** elements for `Microsoft.AspNetCore`, `Microsoft.VisualStudio`, and `Microsoft.EntityFrameworkCore` packages. These tools have been replaced by global tools.
```

Comment:
Are these package name prefixes? If so, let's state that to make it clearer.

Reply:
I'm not sure I've seen a format of ... for example ... `Microsoft.VisualStudio.*`.

Can it just read ...

> Remove all references to **&lt;DotNetCliToolReference&gt;** elements.